README.md

```diff
@@ -105,7 +105,7 @@ To contribute:
 | [Siemens](https://jobs.siemens.com/jobs/318571?lang=en-us&jobPipeline=LinkedIn%28Wrap%29&sourceType=PREMIUM_POST_SITE) | Huntsville, AL OR Longmont, CO| Closed | Software Engineering Intern Co-Op |
 | [Splunk](https://www.splunk.com/en_us/careers/early-talent.html)| San Jose, CA (Remote Option), Vancouver, BC | Open | Software Engineering Co-op (QA, Android, iOS, AR, Front-end, Back-end) |
 | [LinkedIn](https://www.linkedin.com/jobs/search/?f_C=1337&geoId=103644278&keywords=intern%20fall%202022&location=United%20States) | Sunnyvale, CA or San Francisco, CA or New York, NY | Open | Software Engineer, Systems and Infra Engineer (Backend), UI Engineer, Mobile Engineer
-| [UnitedHealthGroup - Optum](https://careers.unitedhealthgroup.com/early-careers-2/programs/technology/) | US | Open | Software Engineering Intern
+| [UnitedHealth Group - Optum](https://careers.unitedhealthgroup.com/early-careers-2/programs/technology/) | Twin Cities, MN, Cypress, CA, Irvine, CA, Schaumburg, IL, Basking Ridge, NJ, Hartford, CT, Raleigh, NC, Boston, MA, Phoenix, AZ, Austin, TX | Open | Software Engineering Intern
```

Comment:
Could we update the locations using the below format:-

Twin Cities - MN, Cypress - CA, Irvine - CA, Schaumburg - IL, Basking Ridge - NJ, Hartford - CT, Raleigh - NC, Boston - MA, Phoenix - AZ, Austin - TX